src/Tizen.TV.UIControls.Forms/GridView.cs

```diff
@@ -0,0 +1,231 @@
+        /// Gets or sets a value that represents the width of an item.
+        /// </summary>
+        public double ItemWidth
+        {
+            get { return (double)GetValue(ItemWidthProperty); }
+            set { SetValue(ItemWidthProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the item's layout alignment along the horizon.
+        /// </summary>
+        public LayoutAlignment ItemHorizontalAlignment
+        {
+            get { return (LayoutAlignment)GetValue(ItemHorizontalAlignmentProperty); }
+            set { SetValue(ItemHorizontalAlignmentProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the item's layout alignment along the vertical.
+        /// </summary>
+        public LayoutAlignment ItemVerticalAlignment
+        {
+            get { return (LayoutAlignment)GetValue(ItemVerticalAlignmentProperty); }
+            set { SetValue(ItemVerticalAlignmentProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the currently selected item from the ItemsSource.
+        /// </summary>
+        public object SelectedItem
+        {
+            get { return GetValue(SelectedItemProperty); }
+            set { SetValue(SelectedItemProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the style of GridView.
+        /// </summary>
+        public string ThemeStyle
+        {
+            get { return (string)GetValue(Specific.StyleProperty); }
+            set { SetValue(Specific.StyleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the value which indicates the direction which items are positioned.
+        /// </summary>
+        public GridViewOrientation Orientation
+        {
+            get { return (GridViewOrientation)GetValue(OrientationProperty); }
+            set { SetValue(OrientationProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the horizontal scroll bar visibility.
+        /// </summary>
+        public ScrollBarVisibility HorizontalScrollBarVisible
+        {
+            get { return (ScrollBarVisibility)GetValue(HorizontalScrollBarVisibleProperty); }
+            set { SetValue(HorizontalScrollBarVisibleProperty, value); }
+        }
+
+        /// <summary>
+        /// Gets or sets the vertical scroll bar visibility.
+        /// </summary>
+        public ScrollBarVisibility VerticalScrollBarVisible
+        {
+            get { return (ScrollBarVisibility)GetValue(VerticalScrollBarVisibleProperty); }
+            set { SetValue(VerticalScrollBarVisibleProperty, value); }
+        }
+
+        /// <summary>
+        /// For internal use.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public void SendItemFocused(GridViewFocusedEventArgs args)
+        {
+            ItemFocused?.Invoke(this, args);
+        }
+
+        public void SendItemSelected(SelectedItemChangedEventArgs args)
```

Comment:
[EditorBrowsable(EditorBrowsableState.Never)]

Reply:
Fixed it. Thank you.